### ISSUE TITLE
Loss plots

### ISSUE THREAD

**Issue by goncayilmaz** [facebookresearch/ov-seg#9]
Hello,

I was wondering if you can provide the loss graphs from W&B. I am trying to train OVSeg myself and I am wondering how the graphs should look like.

Thank you in advance for your response.

**Comment by Jeff-LiangF** [comment]
Hi @goncayilmaz, As the checkpoints were trained half a year ago, unfortunately, I cannot find the W&B logs anymore. Just contact me if you have any questions about replicating the results. ThanksE

**Comment by BKINGING** [comment]
> 你好
> 
> 我想知道您是否可以提供W&B的损失图表。我正在尝试自己训练 OVSeg，我想知道图形应该是什么样子。
> 
> 提前感谢您的回复。

bash /media/hubu/data1/202131116023006_bky/ov-seg-main/open_clip_training/src/scripts/coco_gt_171cls_finetune_VitL.sh
2023-10-01,13：03：34 |资讯 |使用单个进程运行。设备库达：0。
2023-10-01,13：03：34 |资讯 |从OpenAI加载预训练的ViT-L-14。
2023-10-01,13：03：45 |资讯 |型号： 2023-10-01,13：05：
35 |资讯 |完成零镜头图像网。
2023-10-01,13：05：35 |资讯 |评估纪元： 0 ade150-零镜头-val-top1： 0.2742 ade150-零镜头-val-top5： 0.5465
2023-10-01,13：05：35 |资讯 |开始纪元 0
回溯（最近一次调用）：
文件 “/home/hubu/anaconda3/envs/bpy38/lib/python3.8/runpy.py”，第 194 行，在_run_module_as_main
返回_run_code（code， main_globals， None， 文件 “/home/hubu/anaconda3/envs/bpy38/lib/python3.8/runpy.py”， 第 87 行， 在 _run_code
exec（code，
run_globals）
文件 “/media/hubu/Data1/202131116023006_bky/ov-seg-main/open_clip_training/src/training/main.py”， 第 320 行，在 main（）
文件中 “/media/hubu/Data1/202131116023006_bky/ov-seg-main/open_clip_training/src/training/main.py”，第 268 行，在
main train_one_epoch（model， data， epoch， optimizer， scaler， scheduler， ema， args， writer）
文件 “/media/hubu/Data1/202131116023006_bky/ov-seg-main
/open_clip_training/src/training/train.py”，第 69 行，在 train_one_epoch for i
中，批处理枚举（数据加载器）：
文件 “/home/hubu/anaconda3/envs/bpy38/lib/python3.8/site-packages/torch/utils/data/dataloader.py”，第 521 行，在下一个
数据中 = self._next_data（） 文件 “/home/hubu/anaconda3/envs/bpy38/lib/python3.8/site-packages/torch/utils/data/dataloader.py”，第 1203 行，_next_data
返回 self._process_data（data）

文件 “/home/hubu/anaconda3/envs/bpy38/lib/python3.8/site-packages/torch/utils/data/dataloader.py”， 第 1229 行，在
data.reraise（）
_process_data 文件中 “/home/hubu/anaconda3/envs/bpy38/lib/python3.8/site-packages/torch/_utils.py”中，第 434 行，在重新引发
异常
FileNotFoundError： 在 DataLoader 工作进程 0 中捕获 FileNotFoundError。
原始回溯（最近一次调用）：
文件 “/home/hubu/anaconda3/envs/bpy38/lib/python3.8/site-packages/torch/utils/data/_utils/worker.py”，第 287 行，在 _worker_loop data = fetcher.fetch（index）
文件 “/home/hubu/anaconda3/envs/bpy38/lib/python3.8/site-packages/torch/utils/data/_utils/fetch.py”，第 49 行，在 fetch

data = [self.dataset[idx] for idx in possibly_batched_index]
文件 “/home/hubu/anaconda3/envs/bpy38/lib/python3.8/site-packages/torch/utils/data/_utils/fetch.py”，第 49 行，在 data = [self.dataset[idx] for idx in possibly_batched_index] 文件 “/media/hubu/Data1/202131116023006_bky/ov-seg-main/open_clip_training/src/training/data.py”，第 47 行，在
getitem
images = self.transforms（Image.open（str（self.images[idx]
）））
文件 “/home/hubu/anaconda3/envs/bpy38/lib/python3.8/site-packages/PIL/Image.py”，第 3236 行，在 open 中 fp = builtins.open
（文件名， “rb”）
FileNotFoundError： [errno 2] 没有这样的文件或目录： 'coco_gt_171cls/2/000000129379.jpg'

这张照片在数据集中，但找不到错误，还能找到其他一些照片？

**Comment by Jeff001128** [comment]
> Hello,
> 
> I was wondering if you can provide the loss graphs from W&B. I am trying to train OVSeg myself and I am wondering how the graphs should look like.
> 
> Thank you in advance for your response.

Hi @goncayilmaz , have you reproduced the author's results? Could you share your training logs, I'm curious if I did something wrong that I cannot reproduce the result.
